.github/workflows/doc.nuxeo.com.yml

```diff
@@ -0,0 +1,66 @@
+name: Build and Deploy Doc.Nuxeo.com
+
+on: 
+  workflow_dispatch
+#  schedule:
+#    - cron: '0 5 * * *'
+
+jobs:
+  build-and-deploy:
+    runs-on: doc
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '14'
+
+      - name: Check Versions
+        id: check-versions
+        run: |
+          RT="git $(git --version)"
+          RT+=" node version $(node --version)"
+          RT+=" npm version $(npm --version)"
+          RT+=" IP $(curl -s ipinfo.io/ip)"
+          echo "RT=${RT}" >> $GITHUB_ENV
+
+      - name: Save runtime versions
+        run: echo "${RT}" > runtime-versions.txt
+
+      - name: Install dependencies
+        run: npm ci --registry="https://registry.npmjs.org" || npm install --registry="https://registry.npmjs.org"
+
+      - name: Build
+        run: ALGOLIA_KEY=${{ secrets.NX_TOKEN }} DEBUG_FILE_LEVEL='debug' npm start
+
+      - name: Generate Nginx Redirects
+        run: bin/redirects-yml2nginx > redirects.txt
+
+      - name: Create empty redirects_pageid.txt
+        run: echo > redirects_pageid.txt
+
+      - name: Deploy to Server
+        uses: appleboy/ssh-action@v1.0.0
+        with:
+        host: ${{ secrets.DOCTEST_IP }}
+        username: ${{ secrets.DOCTEST_USERNAME }}
+        key: ${{ secrets.DOCTEST_SSH_PRIVATE_KEY }}
```

Comment:
rename variabe so the name is environment agnostic, e.g `DOCTEST_IP` => `SERVER_IP`